src/EprPrnIntegration.Api/UpdateProducersFunction.cs

```diff
@@ -0,0 +1,71 @@
+using EprPrnIntegration.Common.Client;
+using EprPrnIntegration.Common.Constants;
+using EprPrnIntegration.Common.Mappers;
+using EprPrnIntegration.Common.Models;
+using EprPrnIntegration.Common.RESTServices.BackendAccountService.Interfaces;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace EprPrnIntegration.Api;
+
+public class UpdateProducersFunction(IOrganisationService organisationService, INpwdClient npwdClient, 
+    ILogger<UpdateProducersFunction> logger, IConfiguration configuration)
+{
+    [Function("UpdateProducersList")]
+    public async Task Run(
+        [TimerTrigger("%UpdateProducersTrigger%")] TimerInfo myTimer)
+    {
+        logger.LogInformation($"UpdateProducersList function executed at: {DateTime.UtcNow}");
+
+        // Read the start hour (e.g., 18 for 6 PM) from configuration
+        var startHourConfig = configuration["UpdateProducersStartHour"];
+        if (!int.TryParse(startHourConfig, out var startHour) || startHour < 0 || startHour > 23)
+        {
+            logger.LogError(
+                $"Invalid StartHour configuration value: {startHourConfig}. Using default value of 18(6pm).");
+            startHour = 18; // Default to 6 PM if configuration is invalid
+        }
+
+        // Calculate fromDate and toDate
+        var toDate = DateTime.Today.AddHours(startHour); // Configurable hour today
+        var fromDate = toDate.AddDays(-1); // Same hour yesterday
```

Comment:
I think we need to rethink how we will get the fromDate, If any reason jobs doesn't run or failed before pushing data to npwd we will miss the delta of that day run on subsequent run